pkg/scheduler/cache/cache.go

```diff
@@ -69,6 +72,29 @@ type podState struct {
 	bindingFinished bool
 }
 
+type imageState struct {
+	// Size of the image
+	size int64
+	// Used to determine which nodes have this image; the value is a node's name
+	nodes sets.String
```

Comment:
Maybe "A set of node names which have this image present"?

Reply:
Yes that's much better!